ext/standard/basic_functions.stub.php

```diff
@@ -600,50 +600,50 @@ function str_starts_with(string $haystack, string $needle): bool {}
 
 function str_ends_with(string $haystack, string $needle): bool {}
 
-function chunk_split(string $str, int $chunklen = 76, string $ending = "\r\n"): string {}
+function chunk_split(string $string, int $length = 76, string $separator = "\r\n"): string {}
 
-function substr(string $str, int $start, ?int $length = null): string {}
+function substr(string $string, int $start, ?int $length = null): string {}
 
-function substr_replace(array|string $str, array|string $replace, array|int $start, array|int|null $length = null): string|array {}
+function substr_replace(array|string $string, array|string $replace, array|int $start, array|int|null $length = null): string|array {}
 
-function quotemeta(string $str): string {}
+function quotemeta(string $string): string {}
 
 function ord(string $character): int {}
 
 function chr(int $codepoint): string {}
 
-function ucfirst(string $str): string {}
+function ucfirst(string $string): string {}
 
-function lcfirst(string $str): string {}
+function lcfirst(string $string): string {}
 
-function ucwords(string $str, string $delimiters = " \t\r\n\f\v"): string {}
+function ucwords(string $string, string $delimiters = " \t\r\n\f\v"): string {}
```

Comment:
Is there any difference between a "delimiter" and a "separator"?

Reply:
The docs say:
> The optional delimiters contains the word separator characters. 
So, I'd say, no, there is no difference :)

I've also adjusted the csv APIs accordingly (comma *separated* values, not comma delimited values!)